server/command/command.go

```diff
@@ -79,9 +79,43 @@ func GetCommand() *model.Command {
 		AutoComplete:     true,
 		AutoCompleteDesc: "Available commands: subscribe, list, unsubscribe \"<name>\", edit \"<name>\", install cloud/server, help.",
 		AutoCompleteHint: "[command]",
+		AutocompleteData: getAutoCompleteData(),
 	}
 }
 
+func getAutoCompleteData() *model.AutocompleteData {
+	confluence := model.NewAutocompleteData("confluence", "[command]", "Available commands:  subscribe, list, unsubscribe \"<name>\", edit \"<name>\", install cloud/server, help")
```

Comment:
Nit: Is there supposed to be two spaces here?

```suggestion
	confluence := model.NewAutocompleteData("confluence", "[command]", "Available commands: subscribe, list, unsubscribe \"<name>\", edit \"<name>\", install cloud/server, help")
```